packages/proto-loader/src/index.ts

```diff
@@ -115,6 +115,39 @@ export interface EnumTypeDefinition extends ProtobufTypeDefinition {
   format: 'Protocol Buffer 3 EnumDescriptorProto';
 }
 
+export enum IdempotencyLevel {
+  IDEMPOTENCY_UNKNOWN = 0,
+  NO_SIDE_EFFECTS = 1,
+  IDEMPOTENT = 2
+}
+
+export interface NamePart {
+  name_part: string;
+  is_extension: boolean;
+}
+
+export interface UninterpretedOption {
+  name?: NamePart[];
+  identifier_value?: string;
+  positive_int_value?: number;
+  negative_int_value?: number;
+  double_value?: number;
+  string_value?: string;
+  aggregate_value?: string;
+}
+
+export interface CustomHttpPattern {
```

Comment:
Now that you have removed `HttpRule`, `CustomHttpPattern` doesn't seem to be used anywhere

Reply:
Ok, removed `CustomHttpPattern`